src/avt/Pipeline/Data/avtDataAttributes.C

```diff
@@ -2091,47 +2098,16 @@ avtDataAttributes::SetSpatialDimension(int td)
 //  Arguments:
 //      vd       The new variable dimension.
 //
-//  Programmer: Hank Childs
-//  Creation:   September 4, 2001
+//  Programmer: Justin Privitera
+//  Creation:   October 4th, 2023
 //
 //  Modifications:
-//    Kathleen Bonnell, Wed Oct  3 10:57:13 PDT 2001
-//    Add actualData, thisProcsActualData.
-//
-//    Hank Childs, Mon Feb 23 14:19:15 PST 2004
-//    Account for multiple variables.
-//
-//    Kathleen Bonnell, Thu Mar 11 10:32:04 PST 2004 
-//    DataExtents now always have dimension of 1. 
-//
-//    Kathleen Bonnell, Wed Mar 31 08:03:47 PST 2004
-//    Added a reason to the exception.
-//
-//    Hank Childs, Wed Dec  1 15:29:56 PST 2004
-//    Make sure varname is non-NULL, or we'll crash.
-//
-//    Jeremy Meredith, Thu Feb  7 17:52:59 EST 2008
-//    Added component extents for array variables.
 //
```

Comment:
Remove this line. One blank comment line is sufficient.

Reply:
But then there are no blank comment lines under Modifications? I'm confused.

Reply:
Ok, it looks good. Disregard my comment.